### PR TITLE
Hook up animations

### LINKED ISSUE #107: Animation Rigging
Rigging up the client side animations to the animation steps passed in on the turn.

### PR DESCRIPTION
**Pull request by Kenoshen**
Fixes #107

### CODE REVIEW

**Comment by Kenoshen** [comment]
This one's a doozy...

File: client/Fireport/Assets/Scripts/AI/DumbAI.cs [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+                    break;
+            }
+        }
+
+        private void OnDestroy() {
+            WebSocketListener.Instance().Unsubscribe(this);
+        }
+
+        private IEnumerator WaitThenDo(float secondsToWait, Action thenDo) {
+            yield return new WaitForSeconds(secondsToWait);
```
Comment (MondayHopscotch) [inline client/Fireport/Assets/Scripts/AI/DumbAI.cs]: dafuq does `yield` do?

Comment (Kenoshen) [reply client/Fireport/Assets/Scripts/AI/DumbAI.cs]: It is C#'s stupid way of handling some asynchronous stuff.  To be perfectly honest, I don't really *get* it, I just kind of know when I'm suppose to use it in Unity.  To me, it doesn't feel very intuitive at all.  But that is usually the case when there is a feature in a language that doesn't map to any other language you already know.

**Comment by Kenoshen** [comment]
I'm going to go ahead and merge since there is now a PR waiting on this one (#111)

**Pull request merged**